apps/cowswap-frontend/src/modules/limitOrders/pure/DeadlineSelector/deadlines.ts

```diff
@@ -18,7 +18,7 @@ export enum LimitOrderDeadlinePreset {
   ONE_DAY = '1 Day',
   THREE_DAYS = '3 Days',
   ONE_MONTH = '1 Month',
-  SIX_MONTHS = '6 Months (max)',
+  SIX_MONTHS = '1 Year (max)',
```

Comment:
@alfetopito was this intentional? 

I thought it was changed already, and also now it doesn't match te const name and the label

Reply:
Yes!
I believe @fairlighteth did some refactoring where the label was reverted back to an earlier version.
Great catch that I forgot to update the label 🤦 

https://github.com/cowprotocol/cowswap/pull/5282